source/compatibility.tex

```diff
@@ -2004,6 +2004,35 @@
 };
 \end{codeblock}
 
+\diffref{class.copy.elision}
+\change
+A function returning an implicitly movable entity may invoke
+a constructor taking an rvalue reference to a type
+different from that of the returned expression.
+Function and catch-clause parameters can be thrown using move constructors.
+\rationale
+Side-effect of making it easier to write
+more efficient code that takes advantage of moves.
+\effect
+Valid C++17 code may become ill-formed in this International Standard.
```

Comment:
`C++17` -> `\CppXVII{}`.  And we typically say "may fail to compile" rather than "become ill-formed", although I see "become ill-formed" is used in [diff.cpp17.temp] too (it should be fixed as well, separately).

Reply:
Fixed.